### ISSUE TITLE
About datasets

### ISSUE THREAD

**Issue by Joker-ZXR** [CityU-AIM-Group/D2Net#2]
Hi, thank you for open sourcing D2Net, and I have to say that it is indeed very good work!!! I have a few doubts about the split of datasets.
In the paper," We split the training dataset into three folds with 190 sequences as training data and 95 sequences as validation data, then conduct the experiments in three-fold cross-validation manner."
But in the Code, there are  "Testing data". I don't understand where the test data comes from.

In preprocess.py, there are MICCAI_BraTS2018_TrainingData_gz、MICCAI_BraTS2018_ValidationData_gz and MICCAI_BraTS2018_TestingData_gz. How is this divided?
And in split.py, You did a triple cross validation.

I don't quite understand how validation data and testing data are divided. It is so strange.

Looking forward to your reply. Thank you.

**Comment by QiushiYang** [comment]
Sorry for late reply.

The ``training dataset``, ``validation dataset`` and ``testing dataset`` in preprocess.py are official dataset splits of BraTS. We only use the ``training dataset`` as our dataset and manually split it into training set and validation set (for 3 runs towards 3 cross-validation), since the official ``validation dataset`` and ``testing dataset`` does not release segmentation labels.
